apps/builder/app/shared/style-object-model.ts

```diff
@@ -286,6 +286,9 @@ export type ComputedStyleDecl = {
   cascadedValue: StyleValue;
   computedValue: StyleValue;
   usedValue: StyleValue;
+  // We will delete it once we have added additional filters to advanced panel and
+  // don't need to differentiate this any more.
+  listed?: boolean;
```

Comment:
not sure if I should do this or create a separate type

Reply:
Is it used anywhere?

Reply:
couldln't find a better solution for now, everything else results in additional types and overhead